fst/main.py

```diff
@@ -22,7 +24,6 @@ def start_streamlit() -> None:
     subprocess.run(["streamlit", "run", streamlit_app_path])
 
 def start_directory_watcher(path: str, log_queue: multiprocessing.Queue) -> None:
```

Comment:
```suggestion
def start_directory_watcher(path: str, log_queue: multiprocessing.Queue) -> None:
    setup_logger(log_queue)
```

This is needed or else none of the logs show up. 

before

<img width="960" alt="image" src="https://user-images.githubusercontent.com/19176976/229324732-75bcafa6-5770-4d7d-aef4-6b291df45f67.png">


after

<img width="998" alt="image" src="https://user-images.githubusercontent.com/19176976/229324737-db7c27e2-5866-4553-9547-a31626bbb219.png">

Reply:
Show me a screenshot where you're seeing duplicate logs

Reply:
My gut tells me this is a quirk using WSL. See if you can run this on linux using the github codespace.

Reply:
Which logs do you have in mind? 
In the screenshots you're showing different parts of the output in different scenarios, one is failure and one is success.
If it's about 'dbt build' logs (I assume it's this one), then in the second one they're also visible ("Completed successfully").

If it's about the query & build time, they're not visible in case of failure independent of whether 'setup_logger' is added here or not.

Reply:
Here with the _setup_logger()_ in _start_directory_watcher_  ( duplicated output)
![image](https://user-images.githubusercontent.com/9139386/231143360-caaf91ad-ccdd-4793-978c-094283644682.png)
![image](https://user-images.githubusercontent.com/9139386/231143216-95f18b09-1e4f-4406-b33c-c7bc7323da9a.png)

Here without - current state of the PR:
![image](https://user-images.githubusercontent.com/9139386/231143740-e5bd0d31-c578-45a1-a566-d916a95e21bf.png)

Reply:
That's so interesting. I've never seen the duplicate logs before in my setup. This is affirming my point that something may be going on in  WSL

Reply:
double checked in Codespace and the result is similar, instead of triplicating the log output, it's duplicated now (and without the _setup_logger()_ in _start_directory_watcher_ it runs correctly),  super weird. I wonder how it's possible that on Mac you don't get the log outputs without the duplicated logging setup.  
It's being called already in main() which should be enough for the logs to show up. 
Here the screens

![image](https://user-images.githubusercontent.com/9139386/232077117-0bb2cb5b-fc5e-404c-863b-d35b3118fdb5.png)


![image](https://user-images.githubusercontent.com/9139386/232077028-e04eae16-c4c7-4a6a-a32c-1f59b395ef3e.png)

Reply:
I'll check on codespaces once you update the code for my review. I wouldn't be surprised if there's a confounding variable in my local setup that's preventing duplicates logs.